structure/extension/_ATTR.yaml

```diff
@@ -0,0 +1,34 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: structure
+
+uri: https://github.com/glamberson/gedcom-tags/_ATTR
+
+standard tag: _ATTR
```

Comment:
Don't use "standard tag", its for things that have already been standardized.  Just use "extension tags"